### DISCUSSION TITLE
Cylindrical shell example

### DISCUSSION THREAD

**Discussion by hrishiv** [idaholab/moose#19580] [Q&A Modules: Solid mechanics] [answered]
I was performing a simulation with a cylindrical shell model and got some errors so I decided to look into one of the test examples https://github.com/idaholab/moose/blob/next/modules/tensor_mechanics/test/tests/shell/static/pinched_cylinder_symm.i
Pinching of the cylinder is modeled by applying load at the side (1,0,1) in the X direction and the result is correct. Instead, I tried to simulate the same pinching in the Y direction by applying a load at the top (0,1,1) in the Y direction. I expected the displacement at the point of application of the load (0,1,1) in the Y direction in my simulation to be equal to the displacement in the X direction at the point of application of a load (1,0,1) of the example as I just changed the application of pinching force from the side to the top. However, I saw different results. Can someone please help me with this? Thank you in advance.
![image](https://user-images.githubusercontent.com/45951996/145234155-be1cfe01-6f15-4268-9322-9819a925993b.png)

![image](https://user-images.githubusercontent.com/45951996/145234224-c2deaefd-6f00-40f7-a614-a12d6d60cdec.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you please paste your updated DiracKernel for the source?

Guillaume

**Reply by GiudGiud** [reply]
or the whole input file if you have changed anything else

**Reply by hrishiv** [reply]
Hi, I just changed DiracKernel and the postprocessor.
[DiracKernels]
  [./top]
    type = ConstantPointSource
    variable = disp_y
    point = '0 1 1'
    value = -2.5 # P = 10
  [../]
[]

[Postprocessors]
  [./disp_y]
    type = PointValue
    point = '0 1 1'
    variable = disp_y
  [../]
[]

**Reply by GiudGiud** [reply]
This seems right. Not sure why the solution is not symmetric.
Git blame says @sveerara and @bwspenc should know

**Reply by hrishiv** [reply]
Also from what have I observed using shell elements in MOOSE in different orientations has been causing some issues. When I have a model like a box that would have shell elements in two perpendicular planes YZ and XZ, the solution is wrong (compared to ABAQUS) when I specify the same two rotation variables 'rotx' and 'roty' for both the planes. When I divided the model into two blocks and used 'rotz and roty' as the variables for a block in plane YZ and 'rotx and rotz' variables for the block in XZ plane I got displacements similar to ABAQUS for static case (I still have not got similar result for dynamic case using different block too). 
![image](https://user-images.githubusercontent.com/45951996/145496891-6006c031-87af-4e35-a659-255bf709af27.png)

**Reply by GiudGiud** [reply]
@jain651 any thoughts on this?

**Reply by jain651** [reply]
Hrishiv is here at INL as an intern with us. We, including Ben, are looking at this problem together. We will post our findings with this problem as soon as possible.
